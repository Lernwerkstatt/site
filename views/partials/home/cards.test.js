const cards = require("./cards.hbs");

describe("cards.hbs", () => {
  it("should match the snapshot", () => {
    expect(
      cards({
        title: "Aktuelles",
        text:
          "Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis ea consequatur quibusdam sint laboriosam voluptates."
      })
    ).toMatchSnapshot();
  });
});
