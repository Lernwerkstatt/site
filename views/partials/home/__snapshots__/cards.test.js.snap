// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cards.hbs should match the snapshot 1`] = `
"<div class=\\"card easy\\">
  <div class=\\"card-body\\">
    <card-title>
      <h3>Aktuelles</h3>
    </card-title>
    <div class=\\"card-text\\">
      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis ea consequatur quibusdam sint laboriosam voluptates.</p>
    </div>
  </div>
</div>
"
`;
